docs/cli.rst

```diff
@@ -1,11 +1,302 @@
+    4) task: snapshot_changes
+
+
+
+Run Tasks and Flows
+*******************
+
+Execute a specific task or flow with the ``run`` command.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> [options]
+    $ cci flow run <name> --org <org> [options]
+
+This command runs the respective task or flow ``<name>`` against the org ``<org>``. 
+
+.. note::
+
+    You can see a list of available orgs by running ``cci org list``.
+..
+
+    Example: The ``run_tests`` task executes Apex unit tests. Assuming there exists an org named ``dev``, you can run this task against it with the command ``cci task run run_tests --org dev``.
+
+
+
+Troubleshoot Errors
+-------------------
+
+Errors happen! That's why ``cci`` provides tools to extract error details so that they can be reported and triaged.
+
+
+Report Error Logs
+*****************
+
+The ``cci error gist`` command sends the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. For this feature to work you need to make sure that your `GitHub  service is set up with the proper scopes <https://cumu:lusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
+
+The gist includes the:
+
+* Current version of ``cci``
+* Current Python version
+* Path to the Python executable
+* ``sysname`` of the host (such as Darwin)
+* Machine name of the host (such as x86_64)
+* Most recent log file (cci.log) that CumulusCI has created.
+
+The URL for the gist is displayed in the terminal as output, and a web browser automatically opens a tab to the gist.
+
+
+The ``--debug`` Flag
+**********************
+
+All CumulusCI commands can be passed the ``--debug`` flag, so that:
+
+* Any calls to CumulusCI's logger at the debug level are shown.
+* Outgoing HTTP requests are logged.
+* If an error is present, the corresponding stack trace is shown, and the user is dropped into a `post-mortem debugging <https://docs.python.org/3/library/pdb.html#pdb.post_mortem>`_ session.
+
+.. note:: To exit a debugging session, type the command ``quit`` or ``exit``.
+
+
+Log Files
+*********
+
+CumulusCI creates a log file every time a cci command runs. There are six rotating log files (``cci.log, cci.log1...5``) with ``cci.log`` being the most recent. Log files are stored under ``~/.cumulusci/logs`` for Mac and Linux users, and ``C:\Users\yourusername\.cumulusci\logs`` for Windows users.
+
+By default, log files document:
+
+* The last command that was entered by the user.
+* All output from the command (including debug information).
+* If a Python-level exception occurs, the corresponding stack trace.
+
+If you want debug information regarding HTTP calls made during execution, you must explicitly run the command with the ``--debug`` flag set.
+
+.. code-block:: console
+
+    $ cci task run <name> --org <org> --debug
+    $ cci flow run <name> --org <org> --debug
+
+
+View Stack Traces
```

Comment:
I would suggest moving this command up underneath the `cci error gist` section, those two being the commands most likely to be run by an end user who wishes to report a problem, while sections on the log files, the `--debug` flag, and `show_stacktraces` are more for the superuser or CCI developer.

Reply:
@davidmreed I'm a bit confused. Where precisely do you want to post the ``cci flow run<name> --org <org> --debug`` command?

Reply:
l made this change.